roles/kubernetes-apps/cluster_roles/files/k8s-cluster-critical-pc.yml

```diff
@@ -0,0 +1,9 @@
+---
+
+apiVersion: scheduling.k8s.io/v1beta1
+kind: PriorityClass
+metadata:
+  name: k8s-cluster-critical
+value: 1000000000
```

Comment:
This just looks very very wrong to do.

Reply:
The priorityclass here has less priority than the `system-cluster-critical`

Any reason you're not just altering the `system-cluster-critical` and `system-node-critical` to be global ?